### ISSUE TITLE
Question about Upgraded sentiment analysis / not getting exactly the same results 

### ISSUE THREAD

**Issue by TalitaAnthonio** [bentrevett/pytorch-sentiment-analysis#73]
Hi! 

It seems that I am not getting exactly the same reslts as you have in your notebook (codeblock 18) when I run the notebook. I thought that I should be able to get the same results because  you are setting the random seed at the beginning of the notebook. 

The results are similar though: 
<img width="741" alt="Schermafbeelding 2020-05-08 om 15 20 25" src="https://user-images.githubusercontent.com/25078987/81409501-8515ff80-913f-11ea-99ff-5210ba46a395.png">

**Comment by bentrevett** [comment]
This is due to PyTorch having no guarantees about random seeds producing the same results across different versions or even different hardware. See the first paragraph of: https://pytorch.org/docs/stable/notes/randomness.html

It looks like you could potentially reach or even exceed the values of notebook 2 if trained for a few more epochs. I didn't use many epochs in the tutorials due to time, so feel free to use more and see if it helps.

**Comment by TalitaAnthonio** [comment]
Ah I see. Thanks for replying! 

I now notice that I also get similar but different results every time I run the notebook. Could this be because I am using Google Collab (i am also using the gpu there)?

**Comment by bentrevett** [comment]
With Colab you should get the same results every time you do "Restart Runtime and Run all". Let me know if not.

**Comment by TalitaAnthonio** [comment]
Yes you are right!! I just tried it and I get the same results.
